Dockerfile

```diff
@@ -15,12 +15,12 @@
 
 COPY --from=builder /app .
 
-RUN pip install --upgrade pip && \
-    pip install -r requirements.txt
+#RUN pip install --upgrade pip && \
+#    pip install -r requirements.txt
 
-RUN python manage.py migrate
+#RUN python manage.py migrate
```

Comment:
The `RUN` command for running database migrations is commented out. If migrations are necessary during startup, ensure this line is active and that the database is accessible and properly configured.